### PR TITLE
Update microsoft-defender-antivirus-on-windows-server.md

### PR DESCRIPTION
**Pull request by msbemba**
Update navigation to Disable Defender antivirus using group policy

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6572

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-on-windows-server.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ You can now run Microsoft Defender Antivirus in passive mode on Windows Server 2
 
 | Procedure | Description |
 |---|---|
-| Disable Microsoft Defender Antivirus using Group Policy | In your Local Group Policy Editor, navigate to **Windows Defender**, and then select **Turn off Windows Defender Antivirus**. |
+| Disable Microsoft Defender Antivirus using Group Policy | In your Local Group Policy Editor, navigate to **Adiministrative Template**, **Windows Component**, **Endpoint Protectiont**, ** Disable Endpoint Protectiont**, and then select **Enabled** and Click **Ok**. |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-on-windows-server.md]:
```suggestion
| Disable Microsoft Defender Antivirus using Group Policy | In your Local Group Policy Editor, navigate to **Administrative Template** > **Windows Component** > **Endpoint Protection** > **Disable Endpoint Protection**, and then select **Enabled** > **Ok**. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Comment by denisebmsft** [comment]
Thank you @yogkumgit @JohanFreelancer9

**Pull request merged**